manifests/b/BandLab/Cakewalk/29.09.0.098/BandLab.Cakewalk.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: BandLab.Cakewalk
+PackageVersion: 29.09.0.098
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /SP- /SUPPRESSMSGBOXES
+  SilentWithProgress: /SILENT /SP- /SUPPRESSMSGBOXES
+  Custom: /VERYSILENT /SP- /SUPPRESSMSGBOXES
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+UpgradeBehavior: install
+ReleaseDate: 2022-06-23
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.bandlab.com/cakewalk/setup/CakewalkSetup.exe?d=20220627
+  InstallerSha256: E7D75D2E1E1A3BBC6F022F04D12909BBA3AB2898F098A3A27CAADEEE540F8BB2

```

Comment:
This is a bootstrap installer - it will check for the latest version of Cakewalk then download and run that executable. In order to get a silent install, you would need to find the URL for the actual Cakewalk installer and any dependencies it has.

Fortunately, I was able to find the direct URL:
```suggestion
  InstallerUrl: https://downloads.bandlab.com/cakewalk/CbB/29.09.0.098/Cakewalk_by_BandLab_Setup_29.09.0.098.exe
  InstallerSha256: 9D8C2561B4C67A1B39E0CCE8D99D10860F3F214AAE29385F78FEAB9220649465
```

However, I get this error when using `/SUPPRESSMSGBOXES`:
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/b8144877-c99c-481c-875e-e28d894f8e20)

> 2024-05-17 23:31:36.973   Defaulting to Abort for suppressed message box (Abort/Retry/Ignore):
                          C:\Program Files\Cakewalk\Shared Plugins\multivoicechorusflanger.dll

I'm not sure if it's just because I was running it in a sandbox, or if it's truly an error.

Reply:
https://bandlab.github.io/cakewalk/versions.json